src/dmd/dsymbolsem.d

```diff
@@ -6094,7 +6094,7 @@ void templateInstanceSemantic(TemplateInstance tempinst, Scope* sc, Expressions*
     tempinst.parent = tempinst.enclosing ? tempinst.enclosing : tempdecl.parent;
     //printf("parent = '%s'\n", parent.kind());
 
-    TemplateStats.incUnique(tempdecl.isTemplateDeclaration());
+    TemplateStats.incUnique(tempdecl.isTemplateDeclaration(), tempinst);
```

Comment:
I've noticed something when working on the other PR that touches to the same code, why the redundency `tempdecl.isTemplateDeclaration()` ? tempdecl is already known to be a template declaration.

Reply:
Thanks. I'll remove the call.

Reply:
thanks.

Reply:
I'm gonna make dmd detect these mistakes aswell. ;)